PCGen-Formula/code/src/java/pcgen/base/formula/visitor/StaticVisitor.java

```diff
@@ -118,8 +118,7 @@ public StaticVisitor(FunctionLibrary fl)
 	 * 
 	 * {@inheritDoc}
 	 */
-	@Override
-	public Object visit(SimpleNode node, Object data)
+	public Object visit(SimpleNode node)
```

Comment:
This breaks the interface - can't remove this method.  Not sure how the file even compiles with this removed